build.gradle

```diff
@@ -57,9 +57,8 @@ jar {
   }
   exclude 'META-INF/*.RSA', 'META-INF/*.SF','META-INF/*.DSA'
 
-  // Explicitly exclude com/linkedin/data and org/apache/hadoop files from the final jar. They can cause issues in other downstream applications.
+  // Explicitly exclude com/linkedin/data files from the final jar. They can cause issues in other downstream applications.
   exclude 'com/linkedin/data/**'
-  exclude 'org/apache/hadoop/**'
```

Comment:
Is this intended? Do we need the hadoop files in the fat jar?

Reply:
Good point, I'm not sure. It's a test run to validate @jainr 's thought.